tokio-stream/src/macros.rs

```diff
@@ -0,0 +1,18 @@
+macro_rules! cfg_time {
+    ($($item:item)*) => {
+        $(
+            #[cfg(feature = "time")]
+            #[cfg_attr(docsrs, doc(cfg(feature = "time")))]
+            $item
+        )*
+    }
+}
```

Comment:
The `Cargo.toml` does not define a `time` feature.

Reply:
I would consider just having `tokio-stream` hard depend on the `tokio` time feature. It would be temporary. Up to you though.

Reply:
its easy to keep it 🤷 I just added a time feature that is enabled by default.